stumpy/stumpi.py

```diff
@@ -324,7 +324,7 @@ def _update_egress(self, t):
             σ_Q = np.nan
             Q_subseq_isconstant = False
         else:
-            Q_subseq_isconstant = core.rolling_isconstant(
+            Q_subseq_isconstant = core.process_isconstant(
```

Comment:
There is no need to replace `rolling_isconstant` with `process_isconstant` here since the param `T_subseq_isconstant` that is being passed here always has the "function" type. Having said that, I think it is better to use a higher-level API so to be consistent with other parts of the code.

Reply:
sounds good.